indi-duino/devices/Firmwares/weatherradio/weatherradio.ino

```diff
@@ -483,6 +493,11 @@ void loop() {
   updateAnemometer();
 #endif //USE_DAVIS_SENSOR
 
+#ifdef USE_DEW_HEATER
+  // Change this if you want to use a different sensor for the dew heater
+  updateDewheater(dhtData.temperature, dhtData.humidity);
```

Comment:
What about shifting the dependency to the dew heater code like providing a function `updateDewheater()` in the dew heater that is called here and inside this function the call can be called with 
```
void updateDewheater() {
#ifdef USE_DHT
updateDewheater(dhtData.temperature, dhtData.humidity);
#endif // USE_DHT
...
}
```
That's not nice, but this way we keep weatherradio.ino untouched. It is quite lengthy meanwhile and I would not like to force a user to touch it

Reply:
Yep, thats a good solution. I'll find some time to make that change and update the pull. Thanks Wolfgang.